roles/ovirt-engine-setup/README.md

```diff
@@ -0,0 +1,109 @@
+oVirt Engine Setup
+==================
+
+Generate answerfile and run engine-setup with it. In case you pass variable
+'ovirt_engine_update: True' also setup packages will be updated.
+
+Target Systems
+--------------
+
+* engine
+
+Requirements
+------------
+
+ * Preinstalled clean environment with configured repositories.
+ * Ansible version 2.3
+
+Role Variables
+--------------
+
+By default engine-setup uses answer file specific for version of oVirt,
+based on ``ovirt_engine_version`` parameter. You can specify own answer file
+as ``ovirt_engine_answer_file_path``.
+
+| Name                            | Default value         |  Description                                              |
+|---------------------------------|-----------------------|-----------------------------------------------------------|
+| ovirt_engine_type               | ovirt-engine          | Type of product 'ovirt-engine' or 'rhevm'  |
+| ovirt_engine_version            | UNDEF                 | Allowed version: [3.6, 4.0, 4.1] |
+| ovirt_engine_dwh                | UNDEF                 | Bool value for installing local DWH. |
+| ovirt_engine_answer_file_path   | UNDEF                 | /path/to/custom/answerfile. |
+| ovirt_engine_db_host            | localhost             | IP or hostname of PostgreSQL server for engine database. |
+| ovirt_engine_db_port            | 5432                  |  Server listening port. |
+| ovirt_engine_db_name            | engine                | DB name for ovirt-engine. |
```

Comment:
Database name of ovirt-engine.

Not sure but I think `of` is better than `for` here.